tests/helpers.py

```diff
@@ -1111,6 +1111,44 @@ def measure_pvc_deletion_time(interface, pv_name):
     return total.total_seconds()
 
 
+def pod_start_time(pod_obj):
+    """
+
+    Function to measure time taken for container(s) to get into running state
+
+    Args:
+        pod_obj(obj): pod object to measure start time
+
+    Returns:
+        containers_start_time(dict):
+        Returns the name and start time of container(s) in a pod
+
```

Comment:
Remove  spaces please

Reply:
I guess empty lines here is a guideline we follow in this project?